test/src/test/java/org/corfudb/common/metrics/servers/PrometheusMetricsServerTest.java

```diff
@@ -0,0 +1,25 @@
+package org.corfudb.common.metrics.servers;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PrometheusMetricsServerTest {
+    @Test
+    void testConfigParseEmptyOpts() {
+        Map<String, Object> map = new HashMap<>();
+        PrometheusMetricsServer.Config config = PrometheusMetricsServer.Config.parse(map);
+        assertFalse(config.isEnabled());
+    }
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651469)